tumblr/webpack-web-app-manifest-plugin
# Upgrade to Webpack 4 api

# What is this?

[The Webpack 4 plugin API](https://webpack.js.org/api/compiler-hooks/) uses `hookName.tap(...` instead of `plugin(hookName, ...`. This PR switches the plugin over to this new API to eliminate deprecation warnings.

# How to test

Make sure the tests pass! You can also add the plugin to a project and ensure it still produces the expected asset. Here is a screenshot of the version in this PR in use:

![Screen Shot 2019-08-08 at 11 50 57 AM](https://user-images.githubusercontent.com/1395893/62718005-f8cd8500-b9d2-11e9-99d0-7afebf539dab.png)
